src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -278,8 +277,21 @@ public void writeBody(PrintWriter w) {
                             }
                             if (!branchFound) continue;
                             urlFound = true;
-
                             if (!(project instanceof AbstractProject && ((AbstractProject) project).isDisabled())) {
+                                //JENKINS-30178 Add default parameters defined in the job
+                                if (project instanceof Job) {
+                                        List<ParameterValue> jobParametersValues = getDefaultParametersValues((Job) project);
+                                        ListIterator<ParameterValue> buildParameterListIterator = buildParameters.listIterator();
+                                        while (buildParameterListIterator.hasNext()) {
+                                            buildParameterListIterator.next();
```

Comment:
@stephenc It goes through `jobParametersValues` and `buildParameters`, so you add to `buildParameters` the default parameters declared in ` jobParametersValues`. 

* Is this wrong?